views/building-hours/index.js

```diff
@@ -80,7 +86,9 @@ export default class BuildingHoursView extends React.Component {
   refresh = async () => {
     this.setState({refreshing: true})
     await delay(500)
-    this.setState({now: moment.tz(CENTRAL_TZ), refreshing: false})
+    this.setState({now: moment.tz(CENTRAL_TZ), refreshing: false,
+      dataSource: this.getDataSource(),
```

Comment:
Please move the rest of the object to individual lines, like you did with `dataSource`.